survey_form/index.css

```diff
@@ -27,34 +27,106 @@ header::after {
 }
 .nav__link {
   text-decoration: none;
-  color: black;
+  color: #f5f5f5;
   font-family: "Bangers";
   text-transform: uppercase;
   font-size: 25px;
 }
 .nav__link:hover {
-  color: #66cdcc;
+  color: #add8e6;
 }
 .nav__item--logo {
   margin-right: auto;
 }
 .nav__link--logo {
   font-family: "Bangers", cursive;
-  color: #66cdcc;
+  font-size: 30px;
+  color: #add8e6;
 }
 .tittle {
   display: flex;
   justify-content: center;
   margin: auto 0;
 }
-.form {
+.form-survey {
   display: flex;
+  flex-direction: column;
   width: 50%;
   margin: 0 auto;
   background-color: #f5f5f5;
   border-radius: 10px;
+  padding: 1rem;
 }
 
-.form__tittle {
-  margin: 0 auto;
+.form-survey__title {
+  margin-bottom: 1rem;
+  text-align: center;
+}
+
+.form-survey__input {
+  padding: 6px;
+  margin: 0 30% 1rem 30%;
+}
+.form-survey__input:required:invalid {
+  border: 2px solid #f08080;
+  border-radius: 3px;
+}
+
+.form-survey__label {
+  margin: 0.1rem 30% 0.1rem 30%;
+}
+.form-survey__label--radiobutton {
+  margin-bottom: 1rem;
+}
+.form-survey__label--recommendation {
+  margin: -0.2rem 0 0.5rem 1rem;
```

Comment:
Jak pan tutaj zresetuje `margin-bottom` (3 wartość, mnemonik do zapamiętania kolejności: TRBL), to radiobutton będzie się lepiej układał w stosunku do labela (trzeba też wykonać zmiany z innego komentarza): 

Przed:
![checkbox-before](https://user-images.githubusercontent.com/23278550/50047100-7690f900-00af-11e9-88d5-53bcfe71e7ba.png)

Po:
![checkbox-after](https://user-images.githubusercontent.com/23278550/50047104-7db80700-00af-11e9-8f56-c6c81cda5b0d.png)